src/states/Menu.gd

```diff
@@ -26,19 +24,29 @@ func play() -> void:
 	var error := get_tree().change_scene("res://src/states/Game.tscn")
 	assert(not error)
 
+# Go to the multiplayer lobby.
+func go_to_lobby() -> void:
+	var error := get_tree().change_scene("res://src/states/Lobby.tscn")
+	assert(not error)
+
 
 # Create and host a multiplayer session. Triggered by the "Host" button.
 func host_session() -> void:
-	MultiplayerInfo.my_info.name = Global.config.name
-	MultiplayerInfo.my_info.favorite_color = Global.config.favorite_color
+	# The server always has ID 1.
+	var my_info := {
+		"id": 1,
+		"name": Global.config.name
+	}
+	Multiplayer.player_info[1] = my_info
```

Comment:
Move this to after we passed the error check a few lines down, otherwise that player_info might linger around and cause issues in situations like:

1. port is currently blocked
2. server attempts to host on port and fails
3. port is now unblocked
4. server attempts to host again and successes, `multiplayer_info` might be wonky now